sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightPlugin.kt

```diff
@@ -45,6 +45,7 @@ open class SqlDelightPlugin : Plugin<Project> {
 
     var kotlin = false
     var android = false
+    val isMultiplatform = project.plugins.hasPlugin("org.jetbrains.kotlin.multiplatform")
```

Comment:
This can't be checked here. It needs to be in afterEvaluate in case it's applied after our plugin.

Reply:
Currently it's designed so that any order will work which is nice, but we can enforce an order if it breaks other things.